pdns/bpf-filter.cc

```diff
@@ -320,6 +322,25 @@ BPFFilter::BPFFilter(std::unordered_map<std::string, MapConfiguration>& configs,
     throw std::runtime_error("Unsupported eBPF map format, the current internal implemenation only supports the legacy format");
   }
 
+  struct rlimit old_limit;
+  const size_t new_limit_size = 1073741824;
```

Comment:
Please use more readable numbers, like 1024 * 1024 * 1024

Reply:
Yes, that's right. Will make the required changes.